docs/src/common.md

```diff
@@ -194,19 +204,15 @@ For comparison, we now plot the orbit corresponding to the
 solution obtained with the `Vern9()` integration; note that
 the scales are identical.
 ```@example common
-plot(solV, vars=(1, 2), linewidth=1)
+plot(solV, vars=(1, 2), linewidth=1, fmt = :png)
 scatter!([μ, -1+μ], [0,0], leg=false) # positions of the primaries
 xlims!(-1+μ-0.2, 1+μ+0.2)
 ylims!(-0.8, 0.8)
 xlabel!("x")
 ylabel!("y")
 ```
-We note that the orbits do not display the same qualitative features. In particular,
-the `Vern9()` integration displays an orbit which does not visit the secondary,
-as it was the case in the integration using Taylor's method, and stays far
-enough from ``m_1``. The question is which integration should we trust?
-
-We can obtain a quantitative comparison of the validity of both integrations
+We note that both orbits display the same qualitative features. We can obtain a
+quantitative comparison of the validity of both integrations
```

Comment:
I agree that both orbits display the same qualitative properties, but the plots show also qualitative differences, which I  think it  should also be added. For instance, the integration with our package gets closer to the primary  than that with Verrn9.

Reply:
Ok, updated discussion